Update Parser.php

mgralikowski opened the pull request:
According to https://laravel.com/docs/5.8/upgrade#eloquent method's names must be changed.

This change is only for 5.8 and up without backward compatibility. I suggest separate version/branch.

mgralikowski committed:
Update Parser.php

According to https://laravel.com/docs/5.8/upgrade#eloquent method's names must be changed.

This change is only for 5.8 and up without backward compatibility. I suggest separate version/branch.

tripper54 commented:
Note in https://github.com/marcelgwerder/laravel-api-handler/pull/57 I have a fix for this that is backward compatible.

mgralikowski committed:
Update composer.json

Laravel 6.0

mgralikowski committed:
Update ApiHandler.php

Laravel 6

mgralikowski committed:
Update composer.json

L7 support

mgralikowski committed:
Update composer.json

L8

mgralikowski committed:
Update composer.json

mgralikowski committed:
Update Parser.php

Fix PHP 8 error:

array_diff_ukey(): Returning bool from comparison function is deprecated, return an integer less than, equal to, or greater than zero
